index.js

```diff
@@ -17,24 +17,26 @@ module.exports = function(filteredImports) {
 
       ExpressionStatement: {
         enter: function() {
-          shouldRemove.push(false);
+          if (shouldRemove) {
```

Comment:
I was surprised by that too, but somehow that's what Babel traversed. If you remove the guard in the `Identifier` method you can see all the tests failing because it will visit a final Identifier after exiting the Program.

We could also use a subvisitor inside the Program, but I'd like to avoid that to stay somewhat fast.